src/components/dashboard.js

```diff
@@ -1,155 +1,206 @@
 import React from "react";
 import {
-  Container,
   Segment,
   SegmentGroup,
   Header,
-  Checkbox,
-  Label,
-  Table,
   Dimmer,
   Loader,
   Message
 } from "semantic-ui-react";
```

Comment:
if we're using bootstrap, no need of semantic UI, use either of the two. Not both